Dockerfile

```diff
@@ -37,5 +253,7 @@ COPY manifest.json ${FLYWHEEL}/manifest.json
 COPY template.fsf ${FLYWHEEL}/
 
 
+
+
 # Configure entrypoint
-ENTRYPOINT ["/flywheel/v0/run"]
+ENTRYPOINT ["/bin/bash"]
```

Comment:
Double checking that this is correct...?

Reply:
this may change in gears v3 but entrypoints (and docker environments) are ignored for flywheel gears.  I typically set it to bin/bash so that I don't need to include that when I want do `docker run` the image.  Is this best practice?  probably not...I'll change it.